kotlin/android/app/src/main/java/dev/firezone/android/features/session/ui/SessionViewModel.kt

```diff
@@ -23,9 +24,26 @@ internal class SessionViewModel
         val resourcesLiveData: MutableLiveData<List<Resource>>
             get() = _resourcesLiveData
 
-        fun clearToken() = repo.clearToken()
-
+        // Actor name
         fun clearActorName() = repo.clearActorName()
 
         fun getActorName() = repo.getActorNameSync()
+
+        fun addFavoriteResource(id: String) {
+            val value = favoriteResources.value ?: HashSet()
```

Comment:
I don't know much about Kotlin but can't we initialise this with an empty set like `_resourcesLiveData`? That could save us a few lines of code maybe?

Reply:
Done. It does change the Elvis operator into just `!!` but other than that it's still optional. :/